actors/runtime/src/runtime/fvm.rs

```diff
@@ -168,11 +166,9 @@ where
         rand_epoch: ChainEpoch,
         entropy: &[u8],
     ) -> Result<Randomness, ActorError> {
-        Ok(fvm::rand::get_beacon_randomness(
-            personalization,
-            rand_epoch,
-            entropy,
-        )?)
+        // Note: specs-actors treats this as a fatal error.
+        fvm::rand::get_beacon_randomness(personalization, rand_epoch, entropy)
+            .map_err(|e| actor_error!(ErrIllegalArgument; "failed to get randomness: {}", e))
```

Comment:
> Since it beacon randomness involves external systems, a connectivity error there could lead to an error here, without meaning that the argument was invalid.

This method should always return randomness. It'll look back through time to do so.

I see we have a 20 block limit in lotus... but I think that's just arbitrary. I don't believe the chain can even _progress_ without the randomness beacon.

Reply:
Note: the current lotus code will just fail the block if we fail to get randomness.